### DISCUSSION TITLE
[FEATURE REQUEST] Graph feature of math equitions and data without asking the ai model to generate python code

### DISCUSSION THREAD

**Discussion by Linuxuser1234** [open-webui/open-webui#4103] [General]
Please add a feature where you can graph math and data WITHOUT having to ask the ai model to create python code and running it to create the graph 
then first image give me ascii art 
![Screenshot 2024-07-24 172429](https://github.com/user-attachments/assets/581d531d-57a1-4b08-a92a-c5fac2f8bdc5)
![Screenshot 2024-07-24 172631](https://github.com/user-attachments/assets/3b3df2a9-334f-4f1d-b40a-98c5a9bff036)

**Comment by justinh-rahb** [comment]
https://openwebui.com/f/atgehrhardt/wip_artifacts
